rdf-test-suite/common/src/main/scala/org/w3/banana/diesel/DieselOwlPrimerTest.scala

```diff
@@ -0,0 +1,468 @@
+package org.w3.banana.diesel
+
+import org.w3.banana._
+import org.w3.banana.syntax._
+import zcheck.SpecLite
+import scalaz.Scalaz.{ none, some }
+import org.w3.banana.PointedGraphs
+
+/**
+ * This test uses the W3C owl primer example to provide a use case test for Diesel. 
+ *
+ * Owl2:
+ *  - Is the Web Ontology Language
+ *  - Extends the semantics of the description logic SROIQ
+ *  - Has a structural specification that is independent of the concrete exchange syntaxes for OWL 2 ontologies
+ *  - Has a few, W3C defined mappings to exchange syntaxes, including RDF, Functional and Manchester Syntax
+ *
+ */
+class DieselOwlPrimerTest[Rdf <: RDF](implicit ops: RDFOps[Rdf]) extends SpecLite {
+  import ops._
+  import scala.language.postfixOps
+
+  val owl      = OWLPrefix[Rdf]
+  val rdfs     = RDFSPrefix[Rdf]
+  val otherOnt = Prefix("otherOnt", "http://example.org/otherOntologies/families/")
+
+  implicit def toGraph[Rdf <: RDF](s:Iterable[PointedGraph[Rdf]])(implicit ops: RDFOps[Rdf]):Rdf#Graph =
```

Comment:
I remember having to do something similar in some old code. Looks like something we can extract at some point.